docs/source/io_formats/settings.rst

```diff
@@ -923,6 +923,12 @@ attributes/sub-elements:
 
     *Default*: None
 
+  :batches:
+    A list of integers indicating at what batches a surface source file 
+    should be written.
+
+    *Default*: Last batch only
```

Comment:
The default is usually listed as a value for the argument rather than a description. Let's add a sentence above indicating that a source file, `surface_source.h5`, will be generated by default if this attribute is not set.

Reply:
Oh, let's also indicate that the surface source bank will be cleared in simulation memory each time a surface source file is written.

Reply:
Done!